setup.py

```diff
@@ -162,6 +162,8 @@ def is_excluded_dir(dir):
         install_requires=REQUIRES,
         tests_require=TESTS_REQUIRE,
         extras_require=EXTRAS_REQUIRE,
-        packages=find_packages(exclude=EXCLUDE_PATTERNS),
+        packages=find_packages(
+            exclude=["workbench*", "shoop_tests*"] + EXCLUDE_PATTERNS
+        ),
```

Comment:
That does not feel to be correct solution. I'd rather find a way to run Tox so that those are not in the installed package, if that's what we want. There seems to be two different practises for this: including the tests in the package and leaving them out. We may also package them into separate package(s).

See https://pytest.org/latest/goodpractises.html for some thoughts about this.